submissions/ZjeniiaSt/html-css-popup/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>First task. Popup</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/modern-normalize/1.0.0/modern-normalize.min.css"
+      integrity="sha512-ISS7cAi1PEhQ8jnbJpJZMd29NlhNj4AWYyLOSp2CE/CsHxTCu+r+t0D2yoJudVrd0/8fTVPUVDzY5Tvli75u/g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:wght@400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header>
+      <div class="container">
+        <nav class="wrapper">
+          <ul class="nav">
+            <li class="nav-item"><a href="#"></a>Mail</li>
+            <li class="nav-item"><a href="#"></a>Images</li>
+            <li class="nav-item">
+              <input type="checkbox" id="popup" />
+              <label for="popup">
+                <img src="./images/popup-button.png" />
+              </label>
+              <div class="menu">
+                <ul class="menu-list">
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/calendar.png" alt="" /></a
+                    ><span>Calendar</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/mail.png" alt="" /></a
+                    ><span>Mail</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/news.png" alt="" /></a
+                    ><span>News</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/maps.png" alt="" /></a
+                    ><span>Maps</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/files.png" alt="" /></a>
+                    <span>Files</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/contacts.png" alt="" /></a
+                    ><span>Contacts</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/tanslation.png" alt="" /></a
+                    ><span>Translate</span>
+                  </li>
+                  <li class="menu-item">
+                    <a href="#"><img src="./images/videos.png" alt="" /> </a
```

Comment:
Set the equal height to all icons inside the popup.
![image](https://user-images.githubusercontent.com/16196199/182648471-cfe5e323-80ef-4128-98f8-e785427865c5.png)